imager_images/views.py

```diff
@@ -1,3 +1,99 @@
 from django.shortcuts import render
+from imager_images.forms import UserForm, ImagerProfileForm
+# from media import profile_image
+from django.core.urlresolvers import reverse_lazy, reverse
+from models import Photo, Album
+from django.views.generic import CreateView, UpdateView
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.views import redirect_to_login
+from django.shortcuts import redirect
+from dimager.models import ImagerProfile
 
-# Create your views here.
+
```

Comment:
Decide whether you're using the registration package or making your own - it'll be a pain in the ass if you use both.